composer.json

```diff
@@ -31,7 +31,7 @@
         "drupal/domain": "^1.0@beta || ^1.0",
         "drupal/domain_theme_switch": "^1.5",
         "drupal/draggableviews": "^1",
-        "drupal/dropzonejs": "2.1",
+        "drupal/dropzonejs": "2.1 || ^2.5",
```

Comment:
2.2, 2.3, 2.4 are not good?...

Reply:
https://github.com/ymcatwincities/openy/pull/2459#discussion_r643203452
@duozersk

Reply:
I think we can set it to `^2.1` and in project we can chose 2.1, 2.2, 2.3, 2.4, 2.5, 2.6...
Is equivalent to `>=2.1 <3.0.0`